test/Profiler/coverage_relative_path.swift

```diff
@@ -1,16 +1,15 @@
-// %s expands to an absolute path, so to test relative paths we need to create a
-// clean directory, put the source there, and cd into it.
+// To make sure this test is resilient to directory changes, we create nested directories inside of the
+// temporary test directory and assert those exist, or don't exist, in the emitted ir
+//
 // RUN: rm -rf %t
-// RUN: mkdir -p %t/foo/bar/baz
-// RUN: echo "func coverage() {}" > %t/foo/bar/baz/coverage_relative_path.swift
-// RUN: cd %t/foo/bar
+// RUN: mkdir -p %t/root/nested
+// RUN: echo "func coverage() {}" > %t/root/nested/coverage_relative_path.swift
+// RUN: cd %t/root
```

Comment:
Really we only needed 2 levels of nesting to validate we had the absolute path in the output in the first case